### PR TITLE
add module sale_sub_state

### PR DESCRIPTION
**Pull request by chafique-delli**
This module allows to have a sub state in the sale order
for indicate the status of the production or shipping for example.
Two new P.R. to comes with  'sale_mrp_sub_state' and 'sale_stock_sub_state'.

### CODE REVIEW

File: sale_sub_state/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+##############################################################################
+
+{
+    'name': 'Sale Sub State',
+    'version': '0.1',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'description': """
+    This module allows to have a sub state in the sale order
+    for indicate the status of the production or shipping for example.
```
Comment (pedrobaeza) [inline sale_sub_state/__openerp__.py]: s/indicate/indicating

File: sale_sub_state/sale.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+##############################################################################
+from openerp.osv import fields, orm
+
+
+class SaleOrder(orm.Model):
+    _inherit = 'sale.order'
+
+    _columns = {
+        'sub_state': fields.selection(
+            [('', '')],
```
Comment (pedrobaeza) [inline sale_sub_state/sale.py]: This is very ugly. Use a method for populating values.

**Comment by pedrobaeza** [comment]
A better name would be _sale_order_sub_state_, but as is, it doesn't make nothing. Do you plan to add other modules that complement this one?

**Comment by bealdav** [comment]
:+1:

**Comment by yvaucher** [comment]
I disagree with name `sale_order_sub_state`.

However, as this module isn't usable on it's own I would suggest to name it `base_sale_sub_state` as we did it in modules in https://github.com/OCA/carrier-delivery/tree/7.0

**Comment by chafique-delli** [comment]
I agree with name 'base_sale_sub_state' because actually, alone it has 
little interest.

**Comment by bealdav** [comment]
Ok for 'base_sale_sub_state'

**Comment by eLBati** [comment]
LGTM :+1:

**Comment by eLBati** [comment]
Used by https://github.com/OCA/sale-workflow/pull/94 and https://github.com/OCA/sale-workflow/pull/95

File: base_sale_sub_state/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+{
+    'name': 'Base Sale Sub State',
+    'version': '0.1',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'description': """
+    This module allows to have a sub state in the sale order
+    for indicating the status of the production or shipping for example.
+    """,
+    'author': 'Akretion',
```
Comment (yvaucher) [inline base_sale_sub_state/__openerp__.py:31]: Missing OCA is author list

File: base_sale_sub_state/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    'website': 'http://wwww.akretion.com/',
+    'depends': ['sale'],
+    'data': ['sale_view.xml'],
+    'demo': [],
+    'test': [],
+    'installable': True,
+    'active': False,
```
Comment (yvaucher) [inline base_sale_sub_state/__openerp__.py:38]: Please remove this alias to auto_install

**Comment by yvaucher** [comment]
Just 2 smalls remarks in `__openerp__.py` and we are :+1:

**Comment by gurneyalex** [comment]
@chafique-delli can you look at the remarks above, and rebase this PR?

I created #205 to check together the 3 PRs and it shows flake8 errors, which you should fix (assuming you're still interested in these PRs)

**Comment by pedrobaeza** [comment]
Closing as this seems abandoned without repairing comments. If not, please feel free to reopen.

**Pull request closed without merging**